dependencies/aws-sdk-cpp/meson.build

```diff
@@ -0,0 +1,59 @@
+# aws s3 dependency file
+#============================================================================
+
+
+# Look for aws-cpp-sdk, an optional dependency
+#--------------------------------------------------
+aws_sdk_cpp_dep_list = [
+    dependency('aws-cpp-sdk-core', required: get_option('enable_s3')),
```

Comment:
Dumb noob question: is there a recommended number of spaces for indentation?

Reply:
4 spaces tends to be the standard afaict

Reply:
Meson doesn't specify anything, but python knew what it was talking about when it specified  spaces for python code. :P It's a good model to emulate and e.g. I use it in Meson everywhere.